week1/task_判断三角形形状0.1.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h> 
+#include <math.h>
+#include <conio.h>
+#define EPS 1e-1
+int main ()
+{
+	while (1){
+	
+		float a,b,c;
+		printf("请输入三角形的三边长：");
+		scanf("%f%f%f",&a,&b,&c) ;

```

Comment:
Add one space after comma
```suggestion
		scanf("%f%f%f", &a, &b, &c) ;
```